packages/@lwc/engine-core/src/framework/vnodes.ts

```diff
@@ -21,7 +21,7 @@ export type VParentElement = VElement | VCustomElement;
 export type VNodes = Array<VNode | null>;
 
 export interface BaseVNode {
-    type: VNodeType;
+    type: Readonly<VNodeType>;
```

Comment:
Using `Readonly` on an enum doesn't do anything. Did you mean?

```suggestion
    readonly type: VNodeType;
```

Reply:
No, this was just a mistake. The VNodes currently aren't shared, so their members don't need to be `readonly`.